test/bind.js

```diff
@@ -1,36 +1,9 @@
-/* eslint-env node, browser, mocha */
```

Comment:
This autodetected file move is misleading. It should have been `test/test.combokeys.js -> test/index.js`.